### PR TITLE
Chapter probability

### PR DESCRIPTION
**Pull request by manuf3006**

### CODE REVIEW

File: docs/statistics/probability/General.md [hunk 1]
```diff
@@ -61,9 +61,69 @@ Why is probability so important? Probability plays a central role in fields like
...
+???+ question "Task"
+    Now it's your turn to create a random variable. There are several packages in Python that we can use for this purpose. Use the `random` package we already used in the package [management section](../../python/packages.md#standard-library). 
+
+    <figure markdown="span">
+    ![Correlation Types](/assets/statistics/meme_dice.jpg){width=50% }
+    <figcaption>(Source: <a href="https://imgflip.com/i/9599pd">imgflip</a>) </figcaption>
+    </figure>
+
+    1. Now generate your own random number. Use the commands `randint`, `random` and `choices`. A good documentation can be found [here](https://www.w3schools.com/python/module_random.asp)
+    2. Are those numbers really random? Do some research about the `numpy.random.seed` command
```
Comment (JakobKlotz) [inline docs/statistics/probability/General.md:114]:
```suggestion
    2. Are those numbers really random? Do some research about the `random.seed` command
```

Maybe let them look up pseudo-randomness with the `random` module? Then the exercise is all about a single module.
I really like the Real Python articles. That's the [one](https://realpython.com/python-random/#how-random-is-random) I found on this topic.

File: docs/statistics/probability/General.md [hunk 2]
```diff
@@ -75,9 +135,39 @@ In simpler terms, **probability** is typically used to discuss the likelihood of
...
-    In this case, probability is a theoretical expectation, while proportion is based on real, observable outcomes that we can count.
+        # Set overlay mode
+        fig.update_layout(
+            xaxis_title_text='Result',
+            yaxis_title_text='Count',
+            title=dict(
+                    text=f'<b><span style="font-size: 10pt">Experiment: Flipping 20 Coins </span></b>',
+                ),
+            bargap=0.1,
+            #xaxis_range=[-0.5,1.5],
```
Comment (JakobKlotz) [inline docs/statistics/probability/General.md:164]:
```suggestion
```

Remove the comment? 😄

File: docs/statistics/probability/General.md [hunk 3]
```diff
@@ -86,4 +176,396 @@ In simpler terms, **probability** is typically used to discuss the likelihood of
...
+
+        # Set overlay mode
+        fig.update_layout(
+            xaxis_title_text='Result',
+            yaxis_title_text='Probability/Proportion',
+            title=dict(
+                    text=f'<b><span style="font-size: 10pt">Experiment: Flipping 20 Coins </span></b>',
+                ),
+            bargap=0.1,
+            #xaxis_range=[-0.5,1.5])
```
Comment (JakobKlotz) [inline docs/statistics/probability/General.md:245]:
```suggestion
```

File: docs/statistics/probability/General.md [hunk 4]
```diff
@@ -86,4 +176,396 @@ In simpler terms, **probability** is typically used to discuss the likelihood of
...
+                line=dict(color='#E87F2B', width=5),
+                xref='x',
+                yref='y',
+                name='Probability' if i == 0 else None,
+                showlegend=(i == 0) 
+            )
+        fig.show()
+        ```
+
+???+ question "Task"
```
Comment (JakobKlotz) [inline docs/statistics/probability/General.md:263]: They should add the probability as lines to all plots from the previous exercise? So you would end up with 6 plots, right? 🤔

File: docs/statistics/probability/General.md [hunk 5]
```diff
@@ -61,9 +61,69 @@ Why is probability so important? Probability plays a central role in fields like
...
+- The **sample space** is the set of all possible outcomes (heads or tails).
+- The **random variable** assigns a numerical value (e.g., 1 for heads, 0 for tails) to each event.
+
+Thus, random variables enable us to quantify the outcomes of random processes, allowing for further analysis and interpretation.
+
+???+ question "Task"
+    Now it's your turn to create a random variable. There are several packages in Python that we can use for this purpose. Use the `random` package we already used in the package [management section](../../python/packages.md#standard-library). 
+
+    <figure markdown="span">
+    ![Correlation Types](/assets/statistics/meme_dice.jpg){width=50% }
```
Comment (JakobKlotz) [inline docs/statistics/probability/General.md:109]:
```suggestion
    ![Correlation Types](../../assets/statistics/meme_dice.jpg){width=50% }
```

When we're using Markdown we can use relative paths to get rid of these annoying warnings when serving the page:

```
INFO    -  Doc file 'statistics/probability/General.md' contains an absolute link
           '/assets/statistics/meme_dice.jpg', it was left as is. Did you mean
           '../../assets/statistics/meme_dice.jpg'?
```

If it's ok, I'll fix the links in a small commit? 😄

File: docs/statistics/probability/General.md (hunk 1 above)
Comment (manuf3006) [reply docs/statistics/probability/General.md:114]: that's right. i changed the whole chapter from numpy.random to random and i missed this line.

File: docs/statistics/probability/General.md (hunk 4 above)
Comment (manuf3006) [reply docs/statistics/probability/General.md:261]: Basically its one figure. They should use the figure from the example befor and add thos calculated lines.

File: docs/statistics/probability/General.md (hunk 5 above)
Comment (manuf3006) [reply docs/statistics/probability/General.md:109]: that would be nice. thanks

**Comment by JakobKlotz** [comment]
Look's great! I'll use your chapters as a resource from now on 😄
